bin/setup

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+if [ -n "$CI" ]; then
+	yarn global add typings webpack karma typescript
+fi
+
+yarn install
```

Comment:
A lot of this will change with yarn 3, but I want to get GHA -> green -> yarn3 -> node16